### PR TITLE
Make contract constants be properly available

### LINKED ISSUE #371: NNS constants micropackage
## Is your feature request related to a problem? Please describe.
I'm always frustrated when I see "16" duplicated in a lot of places.

## Describe the solution you'd like
Push constants into a micropackage that can be reused from contracts/backend/tests.

### PR DESCRIPTION
**Pull request by roman-khimov**
This allows to not import contracts into other code, RPC bindings are sufficient. Fixes #371.

### CODE REVIEW

File: contracts/netmap/contract.go [hunk 1]
```diff
@@ -332,11 +314,11 @@ func updateCandidateState(ctx storage.Context, publicKey interop.PublicKey, stat
 //
 // UpdateState panics if requested candidate is missing in the current candidate
 // set. UpdateState drops candidate from the candidate set if it is switched to
-// NodeStateOffline.
+// nodestate.Offline.
```
Comment (cthulhu-rider) [inline contracts/netmap/contract.go:317]: mb with square braces then?

**Review by cthulhu-rider — DISMISSED** [review]

File: contracts/netmap/contract.go (hunk 1 above)
Comment (roman-khimov) [reply contracts/netmap/contract.go:317]: Fixed.

**Review by cthulhu-rider — APPROVED** [review]

**Comment by roman-khimov** [comment]
Rebased after #403 merge.

**Review by cthulhu-rider — APPROVED** [review]

**Review by AnnaShaleva — COMMENTED** [review]
Too late, but LGTM.

**Pull request merged**